### PR TITLE
Custom themes, cucumber tests

### PR DESCRIPTION
**Pull request by irenliber**

### CODE REVIEW

File: app/models/admin_user.rb [hunk 1]
```diff
@@ -32,8 +32,10 @@ class AdminUser < ActiveRecord::Base
   validates_format_of :email, with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i, unless:  proc { self.persisted? && email.blank? }
 
   after_save do
-    contact = billing_contact || build_billing_contact
-    contact.update!(email: self.email) if @email
+    unless Rails.env.test?
```
Comment (dmitry-sinina) [inline app/models/admin_user.rb]: Why did you disable it for test environment?

Comment (max-konin) [inline app/models/admin_user.rb]: @dmitry-sinina we have problem with db for test. Only "public" schema works stable

Comment (Fivell) [inline app/models/admin_user.rb]: lets wait for #85, since it is start for CI support in yeti-web

Comment (Fivell) [inline app/models/admin_user.rb]: >> we have problem with db for test. Only "public" schema works stable

@max-konin  can you bring more details please

Comment (max-konin) [inline app/models/admin_user.rb]: @Fivell All schemas except "public" do not exist in DB when specs runing. We are using PG as a docker container with this image https://hub.docker.com/r/maxkonin/yeti_pg/. Mb something wrong with configs.... But I think that all be fine after 85 PR

**Comment by Fivell** [comment]
@irenliber  can you rebase on the top of the master please?

**Comment by max-konin** [comment]
@Fivell @dmitry-sinina I merged this branch with master

**Pull request merged**